packages/node_modules/@webex/plugin-memberships/src/memberships.js

```diff
@@ -549,7 +549,9 @@ const Memberships = WebexPlugin.extend({
   getMembershipEvent(activity, event) {
     try {
       const sdkEvent = cloneDeep(this.eventEnvelope);
-      const cluster = getHydraClusterString(this.webex, activity.target.url);
+      const cluster = (activity.verb !== SDK_EVENT.INTERNAL.ACTIVITY_VERB.HIDE) ?
```

Comment:
This change doesn't seem to match the commit message.

Reply:
New commit messages, plus a new one associated with additional javadocs